core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -200,7 +200,7 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
     }
   }
 
-  val initThread = initialize()
+  var initThread : Thread = null
```

Comment:
sorry nit again: no space between `d` and `:`. Please refer `replayExecutor` for how to space.

Reply:
Done.